packages/clay-data-provider/src/ClayDataProvider.js

```diff
@@ -128,23 +122,23 @@ class ClayDataProvider extends ClayComponent {
 	 */
 	created() {
 		this._isResolvedData = false;
-		this._pollingInterval = 0;
 		if (this._hasData(this.dataSource)) {
 			this._dataSource = this.dataSource;
 			this._isResolvedData = true;
 		} else {
-			this._fetchData();
+			this.updateData('');
 		}
+		this.updateData = debounce(
```

Comment:
I do not know if `debounce` is necessary when we do not want to make a request every time the query changes.